src/docs/product/dashboards/custom-dashboards/index.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Custom Dashboards
+sidebar_order:
+description: "Learn how to edit or create additional dashboards to suit your organization's needs."
+---
+
+<Note>
+
+This feature is available only if your organization is on either a Business or Trial plan.
+
+</Note>
+
+Custom Dashboards are shared across your organization, and users can view each other’s custom dashboards by selecting them from the dropdown menu in the top right. Note that any changes you make in the [default dashboard](/product/dashboards/) (called Dashboard) are global, and will appear for users across the entire organization.
+
+![Dashboard navigation.](dashboard_cropped.png)
+
+Both "Create Dashboard" and "Edit Dashboard" will take you to Edit Mode which will lightly gray out the widgets and present you with several widget and dashboard actions you can take.
```

Comment:
This is a tiny bit misleading since a new dashboard won't show you grayed out widgets. Reworded.

```suggestion
Clicking either "Create Dashboard" or "Edit Dashboard" will take you into edit mode, which allows to edit at both the dashboard and widget level. When you edit an existing dashboard, you can tell that you're in edit mode because the widgets are lightly grayed out.
```

Reply:
Changed